### ISSUE TITLE
I passed " tensorboard_log='logs/PPO' " in the PPO learner in the "train_airl" code, but the folder is empty after trainning, is there something I need to config?

### ISSUE THREAD

**Issue by reneelin1712** [HumanCompatibleAI/imitation#438]
Thanks in advance. 
I have created the logs/PPO folder, so it exists.
I don't know how to read the summary histogram and want to see the generator loss, so I try to plot it in the Tensorboard.  I passed in the argument in the PPO learner.

learner = PPO(
    env=venv,
    policy=MlpPolicy,
    batch_size=32,
    ent_coef=0.0,
    learning_rate=5e-6,
    n_epochs=4,
    n_steps = 512,
    tensorboard_log="logs/PPO",
)
